CentralComputing/Pod.cpp

```diff
@@ -1,33 +1,61 @@
 #include "Sensor_Package.h"
+#include "Pod.h"
 
 #include <thread>
+#include <csignal>
 
 using namespace std;
 
 Sensor_Package * sensors;
+bool running = true;
+
 /**
 * Gets new sensor values from the XMEGA
 */
 void sensor_loop() {
 	//TODO construct XMEGA transfer object
-	//sensors->update();
+	Xmega_Transfer transfer;
+	while(running) {
+		sensors->update(transfer);
+	}
+
 }
 
+void int_handler(int signum) {
+	running = false;
+}
+
+
 int pod(int argc, char** argv) {
 	
+	signal(SIGINT, int_handler);
 	vector<Sensor_Configuration> configs = vector<Sensor_Configuration>();
 
 	//sets up thermocouple configuration
 	Sensor_Configuration thermo;
 	thermo.type = THERMOCOUPLE;
-	thermo.simulation = 0;
-	thermo.count = 4;
+	thermo.simulation = 1;
+	configs.push_back(thermo);
+
+/*
+	Sensor_Configuration accel;
+	accel.type = ACCELEROMETER;
+	accel.simulation = 1;
+	configs.push_back(accel);
+	*/
 
 	//TODO set up other configurations
+
 	
-	sensors = new Sensor_Package(configs);
+	sensors = new Sensor_Package(configs, false);
```

Comment:
It can't just be defined in the sensor_thread because it'll be used in the state thread/server thread to read data.  I'll change it from a global though, that's a good idea